src/Bridge/Doctrine/Orm/Extension/FilterExtension.php

```diff
@@ -27,13 +27,15 @@
  */
 final class FilterExtension implements QueryCollectionExtensionInterface
 {
+    use FilterLocatorTrait;
+
     private $resourceMetadataFactory;
-    private $filters;
 
-    public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFactory, FilterCollection $filters)
+    public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFactory, $filterLocator)
```

Comment:
:+1: I hope that phpstan is fine with it and yeah this is better for future developers IMO. Thanks!